### PR TITLE
Check keys nested

### PR DESCRIPTION
**Pull request by merlokk**
added timeouts for check keys in nested

### CODE REVIEW

File: client/cmdhfmf.c [hunk 1]
```diff
@@ -550,20 +580,23 @@ int CmdHF14AMfNested(const char *Cmd)
...
 		PrintAndLog("d - write keys to binary file dumpkeys.bin");
+		PrintAndLog("s - Slow (1ms) check keys (required by some non standard cards)");
+		PrintAndLog("ss - Very slow (5ms) check keys");
 		PrintAndLog(" ");
 		PrintAndLog("      sample1: hf mf nested 1 0 A FFFFFFFFFFFF ");
 		PrintAndLog("      sample2: hf mf nested 1 0 A FFFFFFFFFFFF t ");
 		PrintAndLog("      sample3: hf mf nested 1 0 A FFFFFFFFFFFF d ");
 		PrintAndLog("      sample4: hf mf nested o 0 A FFFFFFFFFFFF 4 A");
 		PrintAndLog("      sample5: hf mf nested 1 * t");
+		PrintAndLog("      sample5: hf mf nested 1 * ss");
```
Comment (pwpiwi) [inline client/cmdhfmf.c]: sample6

File: client/cmdhfmf.c [hunk 2]
```diff
@@ -580,11 +613,10 @@ int CmdHF14AMfNested(const char *Cmd)
...
 		autosearchKey = true;
 
-		ctmp = param_getchar(Cmd, 2);
-		transferToEml |= (ctmp == 't' || ctmp == 'T');
-		createDumpFile |= (ctmp == 'd' || ctmp == 'D');
+		parseParamTDS(Cmd, 2, &transferToEml, &createDumpFile, &btimeout14a);
 
-		PrintAndLog("--nested. sectors:%2d, block no:*, eml:%c, dmp=%c ", SectorsCnt, transferToEml?'y':'n', createDumpFile?'y':'n');
+		PrintAndLog("--nested. sectors:%2d, block no:*, eml:%c, dmp=%c checktimeout=%d us", 
+			SectorsCnt, transferToEml?'y':'n', createDumpFile?'y':'n', (int)btimeout14a * 10000 / 106);
```
Comment (pwpiwi) [inline client/cmdhfmf.c]: isn't this evaluated right to left? Should be ((int)btimeout14a * 10000) / 106 to minimize rounding errors

**Comment by pwpiwi** [comment]
Please add a line in CHANGELOG regarding the new parameters to hf mf nested

**Comment by merlokk** [comment]
fixed

**Pull request merged**